infrastructure/cluster/flux-v2/reana/reana-release.yaml

```diff
@@ -17,62 +19,40 @@ spec:
       interval: 1m
       version: 0.9.1-alpha.3
 
-
   values:
-  #   debug:
-  #     enabled: false
-  #   eos:
-  #     enabled: false
-  #   maintenance:
-  #     enabled: false
 
     shared_storage: 
       backend: cephfs
-      volume_size: 20
+      volume_size: 1000
       access_modes: ReadWriteMany
       cephfs:
         provisioner: manila-provisioner
         type: "Meyrin CephFS"
         availability_zone: nova
         os_secret_name: os-trustee
         os_secret_namespace: kube-system
-        cephfs_os_share_id: 84395be9-74a1-4000-9a9d-53287a0eb1f4 
-        cephfs_os_share_access_id: 4e165cb2-4960-4129-a0a3-d77f5a606b22
+        # get shares with `openstack share list`, `openstack share show share_id`
+        cephfs_os_share_id: 1c8a0cef-988b-4614-b072-33f75f9a084c
+        cephfs_os_share_access_id: b0a39b58-9b44-4961-b041-53978552970d
     components:
       reana_ui:
-        cern_sso: true
         enabled: true
-        image: reanahub/reana-ui:0.9.1-alpha.3
-      reana_db:
-        enabled: false
-    db_env_config:
-      REANA_DB_NAME: "vre"
-      REANA_DB_PORT: "6600"
-      REANA_DB_HOST: "dbod-vre.cern.ch"
+        cern_sso: true 
+    #   reana_db:
+    #     enabled: false
+    # db_env_config:
+    #   REANA_DB_NAME: "vre"
```

Comment:
Please make sure this is correct, there is no `vre` database. There is one for each app (see https://github.com/vre-hub/vre/wiki/Infrastructure-Management#database), the one for REANA is called `reana`.

Reply:
yes thanks I had realised!